webservices/resources/candidate_aggregates.py

```diff
@@ -97,11 +97,44 @@ def get(self, **kwargs):
         _, query = candidate_aggregate(ScheduleABySize, label_columns, group_columns, kwargs)
         return utils.fetch_page(query, kwargs, cap=None)
 
-
 @doc(
```

Comment:
This doc goes with the resource below - I think we can swap `SCHEDULE_A_STATE_CANDIDATE_TAG` and `SCHEDULE_A_STATE_CANDIDATE_TOTAL_TAG`

Reply:
@lbeaufort thank you!